Documentation for JBD CAN protocol support

dmitrych5 opened the pull request:
Documentation matching the pull request in the dbus-serialbattery repository: https://github.com/mr-manuel/venus-os_dbus-serialbattery/pull/363

Commit:
Documentation for JBD CAN protocol support

mr-manuel commented on docs/general/connect.md:
```diff
@@ -258,6 +258,16 @@ For example, if you are using three batteries (battery 1 dip switches: `1: ON, r
+
+#### CAN
+
+You'll need to change the CAN protocol in the BMS settings to "Victron".
```
Can you specify more detailed on how to do that? Do you need a PC software or is there a Bluetooth app which can be used? How is the app called?

If you set it to Victron, isn't it supported natively by Venus OS?

dmitrych5 replied:
All done! Added answers to these questions to the documentation.

mr-manuel replied:
Thanks!

Commit:
Add more documentation for JBD BMS setup

Commit:
Additional clarifications about JBD CAN support

Commit:
Additional clarifications about JBD CAN support, part 2

Commit:
Updated the correct JBDTools link in the documentation

Commit:
Clearer documentation about JBD bluetooth apps

Commit:
Removed the link to JBDTools
Turns out it shows the protocol selector only for UPxx series, which makes it less likely that any other JBD BMS series supports Victron CAN dialect

mr-manuel commented:
Thank you for helping to improve this driver!
